plugins/input/postgis/postgis_datasource.cpp

```diff
@@ -544,6 +544,8 @@ std::string postgis_datasource::populate_tokens(
                                 bool intersect) const
 {
     std::ostringstream populated_sql;
+    populated_sql << std::fixed;
+    populated_sql.precision(12); // Increase precision tro avoid issues with floating point values
```

Comment:
does it affect all floating point values from that point onwards?

This is the doc I found, TBH I don't know if there's a better reference for C++11 `std`:

http://www.cplusplus.com/reference/sstream/ostringstream/

> Decimal precision for the next floating-point value inserted.

Just curious, I guess you tested it and fits your bill :)

Reply:
It affects all floating point values that use that stream:
```
$ cat prec.cpp 
#include <iostream>
int main()
{
    const double d = 1.2345678901234;
    std::cout << d << '\n';
    std::cout.precision(12);
    std::cout << d << '\n';
    std::cout << d << '\n';
    std::cout << d << '\n';
    std::cout << d << '\n';
}
$ g++ prec.cpp 
$ ./a.out 
1.23457
1.23456789012
1.23456789012
1.23456789012
1.23456789012
```